superset/migrations/versions/3dda56f1c4c6_migrate_num_period_compare_and_period_.py

```diff
@@ -50,6 +51,12 @@ class Slice(Base):
     'hour': 'PT1H',
     'day': 'P1D',
     'week': 'P1W',
+    'week_ending_saturday': 'P1W',
+    'week_start_sunday': 'P1W',
+    'week_start_monday': 'P1W',
+    'week_starting_sunday': 'P1W',
+    'P1W/1970-01-03T00:00:00Z': 'P1W',
+    '1969-12-28T00:00:00Z/P1W': 'P1W',
```

Comment:
This should be the opposite, since it maps the old format to the new one:

```python
'week_ending_saturday': 'P1W/1970-01-03T00:00:00Z',
'week_start_sunday': '1969-12-28T00:00:00Z/P1W',
'week_start_monday': '1969-12-29T00:00:00Z/P1W',
'week_starting_sunday': '1969-12-28T00:00:00Z/P1W',
```